PigLatin/index.test.js

```diff
@@ -0,0 +1,31 @@
+const translate = require("./index");
+
+describe("translate(word)", () => {
+    it("translates 'umbrella' to 'umbrellaway'", () => {
+        expect(translate("umbrella")).toEqual("umbrellaway");
+    });
```

Comment:
These `it` blocks could have a more generic name to support multiple test cases

```suggestion
    it("translates words starting with a vowel'", () => {
        expect(translate("umbrella")).toEqual("umbrellaway");
        expect(translate("equals")).toEqual("equalsway");
        expect(translate("isometric")).toEqual("isometricway");
    });
```